integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -104,6 +106,8 @@ public static void initialize(String appPropsFile) throws Exception {
     }
     username = appProps.getProperty("username", username);
     password = appProps.getProperty("password", password);
+    imageTag = appProps.getProperty("weblogicImageTag");
```

Comment:
Environment variables can override the default value set in props, replace with this
```
weblogicImageTag = System.getenv("IMAGE_TAG_WEBLOGIC") != null ? System.getenv("IMAGE_TAG_WEBLOGIC")  : appProps.getProperty("weblogicImageTag");
weblogicImageName = System.getenv("IMAGE_NAME_WEBLOGIC") != null ? System.getenv("IMAGE_NAME_WEBLOGIC")  : appProps.getProperty("weblogicImageName");
```

Reply:
Done